index.html

```diff
@@ -9,13 +9,17 @@
 	<meta name="author" content="">
 	<link rel="icon" href="../../favicon.ico">
```

Comment:
This is deprecated code from the template we got. 
Could you redirect this to the img folder that you create?